Syslog logging test suite fails

The following test is failing:

```
=== RUN   TestAccFastlyServiceVCL_logging_syslog_useTLS
    block_fastly_service_logging_syslog_test.go:187: Step 1/1 error: Check failed: Check 2/4 error: bad match Syslog logging match:   &fastly.Syslog{
          	... // 9 identical fields
          	Placement:         nil,
          	Port:              &514,
        - 	ProcessingRegion:  nil,
        + 	ProcessingRegion:  &"none",
          	ResponseCondition: &"",
          	ServiceID:         &"jgo44CGDSCjnImLXLJM4lT",
          	... // 8 identical fields
          }
--- FAIL: TestAccFastlyServiceVCL_logging_syslog_useTLS (10.42s)
```

The reason for this is because we set a default in the resource:

```
		"processing_region": {
			Type:         schema.TypeString,
			Optional:     true,
			Default:      "none",
			Description:  "Region where logs will be processed before streaming to BigQuery. Valid values are 'none', 'us' and 'eu'.",
			ValidateFunc: validation.StringInSlice([]string{"none", "us", "eu"}, false),
		},
```

And `Create` unconditionally sets it:

```
ProcessingRegion: gofastly.ToPointer(resource["processing_region"].(string)),
```

--- a/fastly/block_fastly_service_logging_syslog_test.go
+++ b/fastly/block_fastly_service_logging_syslog_test.go
@@ -182,6 +182,7 @@ func TestAccFastlyServiceVCL_logging_syslog_useTLS(t *testing.T) {
 		TLSHostname:       gofastly.ToPointer("example.com"),
 		Token:             gofastly.ToPointer(""),
 		UseTLS:            gofastly.ToPointer(true),
+		ProcessingRegion:  gofastly.ToPointer("none"),
 	}
 
 	resource.Test(t, resource.TestCase{

